CONTRIBUTING.md

```diff
@@ -11,3 +11,12 @@ The goal of the bug report is to explain to us how to reproduce the bug on our c
 * The version of Cauldron
 * Versions of all relevant software, e.g., python, operating systems, etc.
 
+# Suggesting Enhancements
+Before creating enhancement suggestions, please check if that enhancement already exsists. After you've determined which repository your enhancement suggestion is related to, create an issue on that repository and provide the following information:
```

Comment:
There's a typo `exsists` -> `exists`.

Reply:
Also, there's only one repository for Cauldron at the moment so there's no need to determine that.

Reply:
Got it.  Created a new PW with the corrections.